test/vstest.console.PlatformTests/AssemblyMetadataProviderTests.cs

```diff
@@ -109,7 +111,9 @@ public void GetFrameWorkForDotNetAssembly(string framework)
             }
 
             Console.WriteLine("Framework:{0}, {1}", framework, string.Format(PerfAssertMessageFormat, expectedElapsedTime, stopWatch.ElapsedMilliseconds));
-            Assert.IsTrue(stopWatch.ElapsedMilliseconds < expectedElapsedTime, string.Format(PerfAssertMessageFormat, expectedElapsedTime, stopWatch.ElapsedMilliseconds));
+
```

Comment:
if we remove asserts what are we testing?

Reply:
Mayank please remove the commented out code. 

---
In reply to: [165563074](https://github.com/Microsoft/vstest/pull/1406#discussion_r165563074) [](ancestors = 165563074)